docs/server/source/drivers-clients/websocket-event-stream-api.rst

```diff
@@ -0,0 +1,105 @@
+The WebSockets Events Stream API
+================================
+
+BigchainDB provides real-time event streams over the WebSockets protocol with
+the Events Stream API.
+
+Connecting to a Events Stream from your application enables a BigchainDB node
+to notify you as events are processed, such as new `validated transactions <#valid-transactions>`_.
+
+
+Determining Support for the Events Stream API
+---------------------------------------------
+
+In practice, it's a good idea to make sure that the node you're connecting with
+has support for the Events Stream API. To do so, send a HTTP GET request to the
+node's :ref:`Root URL <bigchaindb-root-url>` and check that the response
+contains a ``streams_<version>`` property in ``_links``::
+
+    {
+      "_links": {
+         "streams_v1": "ws://example.com:9984/api/v1/streams/"
+      }
+    }
+
+
+Connection Keep Alive
+~~~~~~~~~~~~~~~~~~~~~
+
+The Events Stream API requires clients to signal that they'd like their
+connection to stay open by sending "pings" across the open connection.
+BigchainDB nodes will automatically close any connections that haven't sent a
+ping in the last three minutes.
+
+.. note::
+
+    While three minutes is the limit before a BigchainDB server will terminate
+    a connection, we suggest sending a ping every 30 seconds for better
```

Comment:
I'd leave them at 30secs for now and then change the spec according to our findings in the implementation and deployment phase.